parachain/scripts/build-docker.sh

```diff
@@ -87,13 +76,43 @@ docker build ${NOCACHE_FLAG} --pull -f ./parachain/docker/Dockerfile \
     --build-arg https_proxy="$PROXY" \
     --add-host=host.docker.internal:host-gateway \
     --network host \
-    --target chain-aio \
-    -t ${GITUSER}/${GITREPO}:${TAG} .
+    --target parachain \
+    --tag ${GITUSER}/${GITREPO}:${TAG} .
 
 # Tag it with latest if no tag parameter was provided
-[ -z "$2" ] && docker tag ${GITUSER}/${GITREPO}:${TAG} ${GITUSER}/${GITREPO}:latest
+[ -z "$TAG" ] && docker tag ${GITUSER}/${GITREPO}:${TAG} ${GITUSER}/${GITREPO}:latest
 
 # Show the list of available images for this repo
 echo "Image is ready"
 echo "------------------------------------------------------------"
-docker images | grep ${GITREPO}
\ No newline at end of file
+docker images | grep ${GITREPO}
+
+# ===================================================================================
+if [ -z "$TAG" ] || [ "$TAG" = "latest" ]; then
```

Comment:
If you want to build all-in-one image for "latest" tag only shouldn't first predicate be removed ?